renderer/plugin.js

```diff
@@ -411,6 +412,13 @@ org.ekstep.questionsetRenderer = IteratorPlugin.extend({ // eslint-disable-line
       }
       return true;
     });
-  }
+  },
+  questionExistInQS: function(savedQSState){
+    if(_.isUndefined(savedQSState)){
+      return false;
+    }else{
+     return _.any(savedQSState.masterQuestionSet, function(item){ return _.isEqual(item.id, savedQSState.currentQuestion.id); })
```

Comment:
This is same as previous logic. Only change I can see, _.contains is replaced with _.any method. 

Update the scenarios in JIRA issue, what you have tested with this logic.

Reply:
As I told, create separate JIRA issue for this. Todays QA deployment you have to share the new JIRA issue with scenarios tested.

Reply:
_.contains will check only array elements, But in this scenario, I have to check an array of objects so I replaced _.contains with _.any method.